packages/frontend-2/lib/viewer/composables/activity.ts

```diff
@@ -81,7 +82,8 @@ export function useViewerUserActivityBroadcasting(
   const apollo = useApolloClient().client
 
   const invokeMutation = async (message: ViewerUserActivityMessageInput) => {
-    if (!isLoggedIn.value) return false
+    const { isEnabled: isEmbedEnabled } = useEmbed()
```

Comment:
Remember that composables can only be invoked 1) in other composables 2) in vue setup functions 3) in nuxt plugins/middlewares.

You're invoking the composable in an arbitrary function - `invokeMutation`. `invokeMutation` can be invoked at any point in time (its triggered manually), so there's no guarantee at all that the `useEmbed()` composable will work.

Move this line a few lines up so that it actually gets invoked at the correct time with the rest of the composables inside `useViewerUserActivityBroadcasting`